ArcGISRuntimeSDKQt_CppSamples/Analysis/TraceUtilityNetwork/TraceUtilityNetwork.cpp

```diff
@@ -1,6 +1,6 @@
-// [WriteFile Name=FindFeaturesUtilityNetwork, Category=Analysis]
+// [WriteFile Name=TraceUtilityNetwork, Category=Analysis]
 // [Legal]
-// Copyright 2019 Esri.
+// Copyright 2020 Esri.
```

Comment:
Existing files do not need to be updated, only new files.

Reply:
Is there a threshold where an existing sample can change so much that we should treat it as a new sample/files? It's basically a shapeshifter and I wasn't sure how to treat it in this situation.

Reply:
No threshold. The determining factor is when the file is committed to source control. So even though this one is renamed, it was still added in 2019 -> remains 2019.